### PR TITLE
GitHub workflows

### PR DESCRIPTION
**Pull request by tecpromotion**
This PR adds workflows and actions.

The following is added.
Greetings: Everyone who adds an issue or PR for the first time will get a thank you.

Furthermore the group joomlagerman/reviewer is added automatically as reviewers :)

### CODE REVIEW

**Comment by tecpromotion** [comment]
@zero-24 please check this pr

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: '👍🎉 You are just super. Thanks for your first Issue on this repo.'
```
Comment (zero-24) [inline .github/workflows/greetings.yml]: Maybe we should choose a german message?

File: .github/auto_assign.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  groupA:
+    - joomlagergerman/reviewer
+
+#  groupB:
+#    - coolwebcreations
+  
+# Set to true to add assignees from different groups to pull requests
+useAssigneeGroups: false
+
+# A list of assignees, split into different froups, to be added to pull requests (GitHub user name)
```
Comment (zero-24) [inline .github/auto_assign.yml]: Lets remove the connected out lines

Comment (tecpromotion) [reply .github/auto_assign.yml]: As a documentation I think that's quite nice and maybe we want to assign it again.

Comment (zero-24) [reply .github/auto_assign.yml]: hmm would't it be better than to add a link to the docs? ;)

Comment (tecpromotion) [reply .github/auto_assign.yml]: done

**Pull request merged**